src/lucky_migrator/create_index_statement.cr

```diff
@@ -0,0 +1,24 @@
+# Builds an sql statement for creating an index using table name, column name,
```

Comment:
an should be `a` in this case

Reply:
Really? I always thought since we pronounce it as "es-ql" that it would be preceded by "an". But I can make that change, for sure.

Reply:
Oh I see what you mean. I say it as "sequel" in which case `a` makes sense, but if you say it spelled out `S.Q.L` then `an` works. Maybe if you capitalize SQL it would make more sense with an? `Build an SQL statement`? Why do you think?

Reply:
Cool, that makes sense to me now. Let's do SQL then, its less ambiguous